main.go

```diff
@@ -2,9 +2,10 @@ package main
 
 import (
 	"fmt"
-	"github.com/urfave/cli"
 	"log"
 	"os"
+
+	"github.com/urfave/cli"
```

Comment:
Looks like this was auto formatted, but I can skip these changes in this PR if you prefer.